### PR TITLE
Fix Rich Presence Nulling

### PR DESCRIPTION
**Pull request by Aluerie**
## Summary

Just a small PR out of #help thread.

## TL;DR for posterity

This PR fixes a bug where `rich_presence` attribute for user classes would reset its value to `None` if the incoming proto doesn't have `rich_presence` field set which occasionally does happen.

## Checklist

- [X] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [X] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Gobot1234** [comment]
Thank you!

File: steam/user.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ class _BaseUser(BaseUser):
 
     def __init__(self, state: ConnectionState, proto: UserProto):
         super().__init__(state, proto.friendid)
+        self.rich_presence = None  # ensure the attr exists
```
Comment (Aluerie) [inline steam/user.py:65]: maybe this should be `self.rich_presence = {}` just so it matches `dict` in typing ? Oups

Comment (Gobot1234) [reply steam/user.py:65]: I think leaving it as None is better because it indicates it didn't come empty from steam?

**Review by Gobot1234 — APPROVED** [review]
